pre_commit/languages/docker.py

```diff
@@ -15,6 +18,32 @@
 healthy = helpers.basic_healthy
 
 
+def is_in_docker() -> bool:
+    try:
+        with open('/proc/1/cgroup') as f:
+            return 'docker' in f.read()
+    except FileNotFoundError:
+        return False
+
+
+def get_docker_path(path: str) -> str:
```

Comment:
probably underscore prefix this and the above function since they are private to this module

Reply:
done

Reply:
done